openedx_ledger/api.py

```diff
@@ -13,6 +13,12 @@ class LedgerBalanceExceeded(Exception):
     """
 
 
+class NonCommittedTransactionError(Exception):
+    """
+    Raised when a transaction that is not in a COMMITTED state is used in a reversal.
+    """
```

Comment:
Also add this to the "Raises:" list of raised exceptions in the function docstring.